projects/sport-data-valley/manual_seeds/daily_responses.rb

```diff
@@ -8,9 +8,9 @@
 module DailyResponses
   extend PeopleHelper
 
-  START_DATE = 10.weeks.ago.beginning_of_day # rubocop:disable Rails/RelativeDateConstant
-  END_DATE = START_DATE + 10.weeks
-  N_PARTICIPANTS = 5
+  START_DATE = 1.week.ago.beginning_of_day # rubocop:disable Rails/RelativeDateConstant
+  END_DATE = START_DATE + 1.week
+  N_PARTICIPANTS = 100
```

Comment:
whenever I have to change this script because  I need to do something very specific to the problem i'm troubleshooting, i make sure to always revert my changes before committing them. Keeping these meaningless changes increases the risk of merge conflicts, and over time would grow the git history unnecessarily. If we find ourselves having to change these parameters too often, then we should consider parameterizing them, if that's even possible.

Reply:
Yeah that makes sense. I'll revert for now.

Reply:
Fixed.